src/test/java/StringTest.java

```diff
@@ -0,0 +1,47 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringTest {
+    
+    @Test
+    @DisplayName("String 클래스에 대한 학습 테스트 : 요구사항 1")
+    public void test1(){
+        String FirstRequirementStringValue  = "1,2";
```

Comment:
변수명을 어떻게 지을까 고민이 많았을 것 같아요.

단순히 input 이라고 사용해봐도 좋을 것 같아요.
또, 재사용 하지 않는 경우라면 인라인으로 처리해도 좋을 것 같아요!

```suggestion
        assertThat("1".split(",")).contains(1)
```